jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -98,7 +98,7 @@ public CRIUSupport(Path imageDir) {
 	 * @return TRUE if support is enabled and the library is loaded, FALSE otherwise
 	 */
 	public static boolean isCRIUSupportEnabled() {
-		return InternalCRIUSupport.isCRIUSupportEnabledAndNativeLoaded();
+		return InternalCRIUSupport.isCRaCorCRIUSupportEnabledAndNativeLoaded();
```

Comment:
This should not return true if crac is enabled

Reply:
line 513 `public synchronized void checkpointJVM() {` should call this method before calling InternalCRIUSupport::checkpointJVM

Reply:
Added `InternalCRIUSupport.isNativeLoaded()`, invoke `InternalCRIUSupport.isCRIUSupportEnabled() && InternalCRIUSupport.isNativeLoaded()` here.